tests/test_datasets/test_dataset.py

```diff
@@ -284,9 +287,14 @@ def test_get_sparse_dataset(self):
         self.assertEqual(len(attribute_names), 20001)
         self.assertTrue(all([isinstance(att, str) for att in attribute_names]))
 
+    @unittest.skipIf(LooseVersion(pd.__version__) < "1.0.0",
```

Comment:
This shouldn't happen, right?

Reply:
yeah, this was done before we removed support so forgot to undo it
will do